tests/Test.ADAL.NET.Unit/BrokerParametersTests.cs

```diff
@@ -83,7 +83,7 @@ public void AcquireTokenInteractiveHandlerConstructor_InitializeBrokerParameters
             Assert.AreEqual(ClientId, brokerParams[BrokerParameter.ClientId]);
 
             Assert.AreEqual(acquireTokenInteractiveHandler.CallState.CorrelationId.ToString(), brokerParams[BrokerParameter.CorrelationId]);
-            Assert.AreEqual(AdalIdHelper.GetAdalVersion(), brokerParams[BrokerParameter.ClientVersion]);
+            Assert.AreEqual(AdalIdHelper.GetAssemblyFileVersion(), brokerParams[BrokerParameter.ClientVersion]);
```

Comment:
Why do we need to change the tests?  Did the behavior of the code for the broker and we did not update the tests yet?

Reply:
The behavior in broker changed and tests were not updated.